src/components/link/link_editor/link_editor.ts

```diff
@@ -98,7 +98,11 @@ export class LinkEditor extends Component<LinkEditorProps, SpreadsheetChildEnv>
   urlInput = useRef("urlInput");
 
   setup() {
-    onMounted(() => this.urlInput.el?.focus());
+    onMounted(() => {
+      const { col, row } = this.props.cellPosition;
+      this.env.model.selection.selectCell(col, row);
```

Comment:
maybe select the cell in `link_display.ts` before opening the component to avoid modifying the state here, which would trigger a double render?

Reply:
Agreed :) change made and update the commit message